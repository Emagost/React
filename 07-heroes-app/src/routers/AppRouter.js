import React from "react";
import {
    BrowserRouter as Router,
    Route,
} from "react-router-dom";
import { Routes } from "../../node_modules/react-router-dom/index";
import { LoginScreen } from "../components/login/LoginScreen";
import { DashboardRoutes } from "./DashboardRoutes";

export const AppRouter = () => {
    return (
        <Router>
            <div>
            <Routes>
                <Route exact path="/login" element={<LoginScreen />} />

                <Route path="/*" element={<DashboardRoutes />} />
            </Routes>
            </div>
        </Router>


    )
}
