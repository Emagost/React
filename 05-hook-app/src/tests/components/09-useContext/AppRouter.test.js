import {shallow} from 'enzyme';
import { AppRouter } from "../../../components/09-useContext/AppRouter";
import UserContext from '../../../components/09-useContext/UserContext';

describe('test <AppRouter/>', () => {
    
    const user = {
        id:1,
        name:'Emanuel'
    }

    const wrapper = shallow(
        <UserContext.Provider value={{
            user
        }}>
            <AppRouter/>
        </UserContext.Provider>
    )

    test('snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    });
    

});
