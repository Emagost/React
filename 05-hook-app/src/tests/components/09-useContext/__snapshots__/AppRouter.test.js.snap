// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test <AppRouter/> snapshot 1`] = `<AppRouter />`;
